ticdc: add mysql sink user privilege (#6072)
pingcap/docs: ticdc/troubleshoot-ticdc.md

cdc cli changefeed update -c test-cf --pd=http://10.0.10.25:2379 --start-ts 415241823337054210
cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
```

> ## When the Sink of replicating data to the downstreams is TiDB or MySQL, what permissions do users of downstream databases need?

Reviewer: ```suggestion
## When the sink of the replication downstream is TiDB or MySQL, what permissions do users of the downstream database need?
```

> When Sink is TiDB or MySQL, the users of downstream databases need the following permissions:

Reviewer: ```suggestion
When the sink is TiDB or MySQL, the users of the downstream database need the following permissions:
```

- `Select`
- `Index`
- `Insert`
- `Update`
- `Delete`
- `Create`
- `Drop`
- `Alter`
- `Create View`

> If you need to replicate `recover table` to the downstream TiDB, `Super` permission is required.

Reviewer: ```suggestion
If you need to replicate `recover table` to the downstream TiDB, the `Super` permission is required.
```
